source/documentation/runbooks/dns/decommission-dns.html.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Decommissiomning Domains
+last_reviewed_on: 2023-01-25
+review_in: 3 months
+---
+
+# Decommissiomning Domains
+
+## Overview
+
+It's important that decommssioning of a domain is part of decommissioning process for any service. 
```

Comment:
typo decommssioning should be decommissioning

Reply:
Fixed typo